plugins/CoreAdminHome/Tasks.php

```diff
@@ -133,6 +143,36 @@ private function rememberTrackingCodeReminderRan($idSite)
         $table->save($settings);
     }
 
+    /**
+     * To test execute the following command:
+     * `./console core:run-scheduled-tasks "Piwik\Plugins\CoreAdminHome\Tasks.cleanupTrackingFailures"`
+     *
+     * @throws \Exception
+     */
+    public function cleanupTrackingFailures()
+    {
+        // we remove possibly outdated/fixed tracking failures that have not occurred again recently
+        $this->trackingFailures->removeFailuresOlderThanDays(Failures::CLEANUP_OLD_FAILURES_DAYS);
+    }
```

Comment:
If this deletes tracking failures that are older than 2 days, and the notify task is run weekly, then that means the email will never report failures or not report all failures (since they can be removed before being sent off). Is that correct or am I missing something?

Reply:
It would report failures that happened in the last 2 days before the email was sent. Eg if email is sent on Sunday at 8am, it would report failures that happened between Friday 8am and Sunday 8am. But not any problems before that... the assumption is that those problems have been fixed basically, or occurred maybe only once but are not a permanent problem.

Reply:
I guess for tracking code issues it's probably not a big deal, people are like invested in making sure they work at first, and then if they break later, it will be a consistent problem. So makes sense to me.